internal/feeder/rekor/rekor_feeder.go

```diff
@@ -65,14 +80,24 @@ func FeedLog(ctx context.Context, l config.Log, w feeder.Witness, c *http.Client
 		if err := getJSON(ctx, c, lURL, "api/v1/log", &li); err != nil {
 			return nil, fmt.Errorf("failed to fetch log info: %v", err)
 		}
-		return []byte(li.SignedTreeHead), nil
+		// Active shard
```

Comment:
Not a blocker, but I think this means that for every shard in the config, we're going to re-request the same JSON from the loginfo endpoint and then pull out just the bit for the relevant shard.

Might be worth popping a comment to this effect, or perhaps a `TODO` to investigate (at some point in the future) whether we can, or even should, improve the efficiency by fetching once per sigstore instance?
It may very well not be worth the lift in complexity, or the special-casing which would be necessary for the config, etc. but having the note in there will help future us when we come across this again :)

Reply:
Added a comment. Another option would be to design this such that it automatically pulls in every shard - This is something I'm exploring in sigstore/rekor-monitor.